docs/development/resources/c++.md

```diff
@@ -0,0 +1,13 @@
+# C++ Resources
+
+Looking at the electron source code without context can be overwhelming. Here are a list of some objects, and links about those objects. Not intended to be comprehensive, and not all may be necessary to contribute to electron. Familiarity with these objects has been helpful for me.
```

Comment:
s/electron/Electron

Reply:
If these docs are to be used by consumers, we'll also want to remove personal references like "Familiarity with these objects has been helpful for me."